dev/planning-example.py

```diff
@@ -0,0 +1,206 @@
+obs.pressure = 0.600 * u.bar
+obs.relative_humidity = 0.2
+obs.temperature = 10 * u.deg_C
+
+# Dates are assumed to be UTC by default
+
+"""
+We can ask about typical times of interest at this observing position
+Returned dates are assumed to be in the observer's time zone for convenience.
+The date parameter, if given, can be an astropy Time or a datetime instance.
+If no parameter is given, the current date/time is assumed.
+"""
+
+# Define a time
+from astropy.time import Time
+time_obs = Time(2457189.500000, format='jd')
+
+# sunset
+obs.sunset(time=time_obs, which='nearest') # Default
+obs.sunset(time=time_obs, which='next')
+obs.sunset(time=time_obs, which='previous')
+obs.sunset() # Will be use time=Time.now(), which='nearest'
+
+# sunrise
+obs.sunrise(time=time_obs, which='nearest') # Default
+obs.sunrise(time=time_obs, which='next')
+obs.sunrise(time=time_obs, which='previous')
+
+# moon rise
+obs.moonrise(time=time_obs, which='nearest')
+
+# moon set
+obs.moonset(time=time_obs, which='nearest')
+
+# The above functions can be called with an `angle` keyword argument to specify
+# a particular horizon angle for rising or setting, or can be called with
+# convenience functions for particular morning/evening twilight.
+# For example, to compute astronomical twilight by specifying the `angle`:
+obs.sunset(time=time_obs, which='next', angle=18*u.degree) 
+
+# evening (astronomical) twilight
+obs.evening_astronomical(time=time_obs)
+
+# evening (nautical) twilight
+obs.evening_nautical(time=time_obs)
+
+# evening (civil) twilight
+obs.evening_civil(time=time_obs)
+
+# morning (nautical) twilight
+obs.morning_nautical(time=time_obs)
+
+# morning (civil) twilight
+obs.morning_civil(time=time_obs)
+
+# morning (astronomical) twilight
+obs.morning_astronomical(time=time_obs)
+
+# what is the moon illumination?
+# returns a float, which is percentage of the moon illuminated
+obs.moon_illumination(time=time_obs)
+
+# what is the moon altitude and azimuth?
+obs.moon_altaz(time=time_obs)
+
+# Other sun-related convenience functions:
+obs.noon(time=time_obs, which='nearest')
+obs.midnight(time=time_obs, which='nearest')
+
+# ==============
+# Target objects
+# ==============
+'''
+A target object defines a single observation target, with coordinates
+and an optional name.
+'''
+from astroplan import FixedTarget, airmass_plot
+
+# Define a target.
+
+from astropy.coordinates import SkyCoord
+t1 = FixedTarget(name='Polaris',
+                 coord=SkyCoord('02h31m49.09s', '+89d15m50.8s', frame='icrs'))
+
+# Leaves scope for NonSiderealTarget, etc.
+
+# Convenience methods for looking up/constructing targets by name via
+# astroquery:
+t1 = FixedTarget.from_name('Polaris')
+
+# ================================
+# Condition objects, observability
+# ================================
+
+"""
+Q: Can I observe this target on the night of May 1, 2015 between 18:30
+and 05:30 HST, above airmass 1.2, on a telescope that can observe between 
```

Comment:
To close the loop on this, a recent change made this `AirmassRange`, but it can still be used in the single-argument form to mean "at this airmass or better".